andela/travel_tool_front
# #161045193 Enable travel admin change requester room

#### What does this PR do?
Updates notification pane to display "View Details" link on changed room notifications

#### Description of Task to be completed?
- Update NotificationContainer to include changed room message to enable the "View Details" link show

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Enter into the project directory - `cd travel_tool_front`
- checkout the branch `git checkout ft-change-room-161045193`
- Start the server with `make start`
- Start the backend of this project `git clone https://github.com/andela/travel_tool_back.git && make start`
- Visit https://travela-local.andela.com
- Login to the application
- Ensure you have a travel admin and manager privilege
- Create a request and approve it.
- Navigate to the residence manage page and click on a guesthouse with bookings
- Click on any booking record and change the allocated room to another
- You should get a notification as a requester once the change is made
- Navigate to the notification pane and click on the "view details" link on a notification items
- You should be redirected to the request details page.


#### Any background context you want to provide?
- You should have a few request on the system so you can view the booked rooms for the requests.
- Once a room is booked, it becomes unavailable whether the manager has approved it or not.
- You cannot change requester to a new room if it is occupied by an opposite gender

#### What are the relevant pivotal tracker stories?
[#161045193](https://www.pivotaltracker.com/story/show/161045193)

#### Screenshots (if appropriate)
![notification-demo](https://user-images.githubusercontent.com/13545380/48268081-53ca4f80-e434-11e8-85bc-813771ab4389.gif)


#### Questions:
